### DISCUSSION TITLE
Safari on iPhone Misidentifies Password Field as New Password - Solution

### DISCUSSION THREAD

**Discussion by this-xkit** [wg-easy/wg-easy#1255] [Ideas]
Hello everyone,

I encountered an issue while using WG-Easy on my iPhone through Safari. The password input field on the login page was being misidentified by Safari as a field for creating a new password, prompting the browser to offer to generate a secure password instead of filling in the existing one.

After some investigation, I found that the problem can be fixed by adding the autocomplete="current-password" attribute to the password input field in the index.html file. This small change tells Safari that the field is for entering an existing password, which allows the browser to correctly offer saved passwords for autofill.

Steps to Fix:
1. Open the index.html file located in src/www/.
2. Locate the password input field:

```
<input type="password" name="password" :placeholder="$t('password')" v-model="password"
    class="px-3 py-2 text-sm dark:bg-neutral-700 text-gray-500 dark:text-gray-500 mb-5 border-2 border-gray-100 dark:border-neutral-800 rounded-lg w-full focus:border-red-800 dark:focus:border-red-800 dark:placeholder:text-neutral-400 outline-none" />
```

4. Modify the input field by adding autocomplete="current-password":

```
<input type="password" name="password" autocomplete="current-password" :placeholder="$t('password')" v-model="password"
    class="px-3 py-2 text-sm dark:bg-neutral-700 text-gray-500 dark:text-gray-500 mb-5 border-2 border-gray-100 dark:border-neutral-800 rounded-lg w-full focus:border-red-800 dark:focus:border-red-800 dark:placeholder:text-neutral-400 outline-none" />
```

6. Save the changes and reload the page.

**Comment by tetuaoro** [comment]
Was added #1249
